data/scripts/creaturescripts/bestiary_kills.lua

```diff
@@ -0,0 +1,47 @@
+local PARTY_MEMBERS_ENABLED = false
```

Comment:
Any reason not to keep it in config.lua?

Reply:
There is no reason, maybe the idea of ​​having the boolean in config.lua is a good idea to maintain consistency with other systems